androidanalyticsdebugger/src/main/java/app/avo/androidanalyticsdebugger/debuggereventslist/DebuggerEventsListAdapter.java

```diff
@@ -239,6 +254,21 @@ public List<DebuggerEventItem> getExpendedEvents() {
         return expendedEvents;
     }
 
+    public SortedList<DebuggerEventItem> getEvents() {
+        return events;
+    }
+
+    void filter(@NotNull String s) {
```

Comment:
Do the filters persists if the debugger is collapsed and opened again? I can imagine that being a common use case